ament_tools/build_type.py

```diff
@@ -18,7 +18,9 @@
 
 
 class BuildAction(object):
-    """Represents an action to do at build time, either a command or a functor.
+
+    """
```

Comment:
Docstrings should always start on the line with the quotes. both single and multi line https://www.python.org/dev/peps/pep-0257/#id17